libretro_core_options.h

```diff
@@ -405,7 +405,7 @@ struct retro_core_option_definition option_defs_us[] = {
    {
       BEETLE_OPT(image_offset),
       "Offset Cropped Image",
-      "When 'Crop Overscan' is enabled, allows the resultant cropped image to be offset horizontally by the specified number of pixels. May be used to correct alignment issues.",
+      "When 'Crop Overscan' is enabled, allows the resultant cropped image to be offset horizontally to the right by the specified number of pixels. May be used to correct alignment issues. As with 'Crop Overscan', currently only supported by the software renderer.",
```

Comment:
'to the right' - This is wrong. The offset can be positive (to the right) or negative (to the left).

Reply:
The reason why I put 'to the right' is because the original description doesn't specify if a positive offset is shifting the image to the right or to the left. I assumed that adding 'to the right' would make it more obvious that a positive shift would be to the right and a negative shift 'to the right' would move the image left.

Maybe something like "When 'Crop Overscan' is enabled, allows the resultant cropped image to be offset horizontally to the right (if offset is positive) or to the left (if number of pixels is negative) . . ." would be less ambiguous?